readme.md

```diff
@@ -53,463 +45,452 @@ html
+
+```js
+$("#red").remove();
 ```
+
 <details>
-<summary>
-Vanilla Javascript syntax
-</summary>
-```javascript
-.removeChild()
-```
+  <summary><strong>Vanilla Javascript syntax</strong></summary>
+
+  ```js
+  .removeChild()
+  ```
+
 </details>
 
 #### `.empty()`
-- removes all the child elements of the jquery object it is called on
 
-```javascript
-$( "ul" ).empty();
+Removes all the child elements of the jQuery object it is called on.
+
+```js
+$("ul").empty();
 ```
 
-### Others (5 minutes)
+### Others (5 minutes / 2:10)
 
 #### `.hide()`
-changes elements style to have `display:none`
 
- ```javascript
- $( "#red" ).hide();
- ```
+Changes elements style to have `display: none`
+
+```js
+$("#red").hide();
+```
 
 #### `.show()`
-changes a `display:none` to `display:block` or whatever it initially was
+
+Changes a `display: none` to `display: block` or whatever it was initially.
 
 ```html
-<li id ="red" style ="display:none">red</li>
+<li id="red" style="display: none;">red</li>
 ```
 
-```javascript
-$( "#red" ).show();
+```js
+$("#red").show();
 ```
 
-#### You Do: Document Dive and examples (10 minutes)
+#### You Do: Document Dive (10 minutes / 2:20)
 
-For the remaining three methods, find a partner, research and provide an example for each. Be prepared to share your answer.
+> 5 minutes exercise. 5 minutes review.
 
-   `.addClass()`
-   [documentation](http://api.jquery.com/addClass/)
+For the remaining three methods, find a partner, research and provide an example for each. Be prepared to share your answer.
 
-   `.removeClass()`
-   [documentation](http://api.jquery.com/removeClass/)
+- `.addClass()` ([documentation](http://api.jquery.com/addClass/))
+- `.removeClass()` ([documentation](http://api.jquery.com/removeClass/))
+- `.toggleClass()` ([documentation](http://api.jquery.com/toggleClass/))
 
-   `.toggleClass()`
-   [documentation](http://api.jquery.com/toggleClass/)
+## You do: Logo Hijack (15 minutes / 2:35)
 
-## You do: Logo hijack (15 min)
+1. Open up the [Microsoft website](https://www.microsoft.com/en-us/) in Chrome or Firefox, and open up the console
+2. Find an image url for the apple logo
+3. Store the url to the apple logo in a variable
+4. Find the Microsoft logo using Javascript and store it in a variable
+5. Modify the source of the logo image so that it's an Apple logo instead
+6. Find the Microsoft search input and store it in a variable
+7. Modify the placeholder of the input so that it says "Search Apple" instead
 
-1. Open up https://www.microsoft.com/en-us/ in Chrome or Firefox, and open up the console.
-1. find an image url for the apple logo
-1. Store the url to the apple logo in a variable.
-1. Find the Microsfot logo using JS and store it in a variable.
-1. Modify the source of the logo IMG so that it's an apple logo instead.
-1. Find the Microsoft search input and store it in a variable.
-1. Modify the placeholder of the input so that it says "Search Apple" instead.
+#### Bonus
 
```

Comment:
`.each` shouldn't be bonus content. Think you should definitely try to find time for this in the lesson.